homeassistant/components/nws/__init__.py

```diff
@@ -128,6 +148,17 @@ async def update_forecast_hourly() -> None:
     await coordinator_forecast.async_refresh()
```

Comment:
For the future: Sometimes its faster to do an `asyncio.gather` here (with `create_eager_task`)

It will depend on if more connections is slower or faster with the remote server
example https://github.com/home-assistant/core/blob/189c07d502fe67e62d031dd6c4088ef259e2e351/homeassistant/bootstrap.py#L430

Reply:
If I had to guess, NWS won't like multiple connections, but I can look into testing.